product_supplierinfo_for_customer_sale/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+    :target: http://www.gnu.org/licenses/agpl-3.0.en.html
+
+==========================================
+Product supplier info for customers - Sale
+==========================================
+
+This module extends of module *Product supplier info for customers* and allows
+use the product codes and product name configured in each products in sale
+orders.
+
+Installation
+============
+
+To install this module, you need:
+
+- Not special pre-installation is required, just install as a regular Odoo
+  module:
+
+  - Go to ``Settings > Module list`` search for the current name and click in
+    ``Install`` button.
+
+Usage
```

Comment:
Or better, explain how to use it
And add a image with the result from sale.order
 - sale_order.product_id: [internal code] internal product
 - sale_order.name: [Customer code] Customer Name